Description: Update default file locations to reflect packaging.

Index: heating-controller-0-164-g3da44c6/main.go
===================================================================
--- heating-controller-0-164-g3da44c6.orig/main.go
+++ heating-controller-0-164-g3da44c6/main.go
@@ -19,6 +19,6 @@
 const (
-	defaultConfigFile = "./config.json"
-	defaultDataDir    = "./data"
+	defaultConfigFile = "/etc/heating-controller.json"
+	defaultDataDir    = "/var/lib/heating-controller/data"
 	defaultPort       = 8080
-	templateDir       = "webserver/templates"
+	templateDir       = "/usr/share/heating-controller/templates"
 )
