Description: Fix spelling error detected by lintian
Author: Nick Morrott <knowledgejunkie@gmail.com>
Last-Update: 2017-01-24
---
--- a/filter/tv_imdb
+++ b/filter/tv_imdb
@@ -151,7 +151,7 @@
 Needs some more controls for fine tuning "close" matches. For
 instance, currently it looks like the North America grabber only has
 date entries for movies, but the imdb.com data contains made for video
-movies as well as as real movies, ot is itE<39>s possible to get the
+movies as well as as real movies, so itE<39>s possible to get the
 wrong data to be inserted. In this case we may want to say "ignore tv
 series" and "ignore tv mini series".  Along with this, weE<39>d want
 to define what a "close" match is. For instance does a movie by the
