Description: Makefile respects DESTDIR for install
Author: Alex Tomlins <alex@tomlins.org.uk>
--- a/Makefile
+++ b/Makefile
@@ -35,13 +35,13 @@ $(EXENAME): mcrcon.c
 ifneq ($(OS), Windows_NT)
 .PHONY: install
 install:
-	$(INSTALL) -vD $(EXENAME) $(PREFIX)/bin/$(EXENAME)
-	$(INSTALL) -vD -m 0644 mcrcon.1 $(PREFIX)/share/man/man1/mcrcon.1
+	$(INSTALL) -vD $(EXENAME) $(DESTDIR)$(PREFIX)/bin/$(EXENAME)
+	$(INSTALL) -vD -m 0644 mcrcon.1 $(DESTDIR)$(PREFIX)/share/man/man1/mcrcon.1
 	@echo "\nmcrcon installed. Run 'make uninstall' if you want to uninstall.\n"
 
 .PHONY: uninstall
 uninstall:
-	rm -f $(PREFIX)/bin/$(EXENAME) $(PREFIX)/share/man/man1/mcrcon.1
+	rm -f $(DESTDIR)$(PREFIX)/bin/$(EXENAME) $(DESTDIR)$(PREFIX)/share/man/man1/mcrcon.1
 	@echo "\nmcrcon uninstalled.\n"
 endif
 
